packages/core/test/popover/popoverTests.tsx

```diff
@@ -33,13 +46,13 @@ describe("<Popover>", () => {
         testsContainerElement.remove();
     });
 
-    describe("validation:", () => {
+    describe.skip("validation:", () => {
```

Comment:
use `{ expectPropValidationError } from "@blueprintjs/test-commons";`
fine to do in a follow-up PR, just make a note in this PR description